src/test/java/org/opensearch/security/securityconf/SecurityRolesPermissionsTest.java

```diff
@@ -95,6 +106,25 @@ static ObjectNode role(final String... clusterPermissions) {
             .set("cluster_permissions", clusterPermissionsArrayNode);
     }
 
+    static ObjectNode roleWithIndexPermission(final String[] allowedActions) {
```

Comment:
is this meant for regular index permissions? how does this method differ from roleWithIndexPermission?